bin/runtests

```diff
@@ -182,14 +189,15 @@ function show_help()
 
 <comment>Options:</>
 
-  -i, --initialize          Execute the initializaction script before running the tests.
-  -f, --flags               Pass flags to PHPUnit.
-  -t, --target              Specify a target bundle.
-  -a, --all                 Run all tests.
-  -r, --restart-jackrabbit  Restart jackrabbit between bundle tests.
-  -h, --help                Show this help.
-  -C, --no-component-tests  Do not run the component tests.
-  -B, --no-bundle-tests     Do not run the bundle tests.
+  -i, --initialize            Execute the initializaction script before running the tests.
+  -f, --flags                 Pass flags to PHPUnit.
+  -t, --target                Specify a target bundle.
+  -a, --all                   Run all tests.
+  -r, --restart-jackrabbit    Restart jackrabbit between bundle tests.
+  -h, --help                  Show this help.
+  -C, --no-component-tests    Do not run the component tests.
+  -B, --no-bundle-tests       Do not run the bundle tests.
```

Comment:
Maybe, we could probably refactor this to be much better, but out of scope here I think.